service/templates/admin.html

```diff
@@ -458,6 +458,32 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 									</table>
 
 								</md-card-content>
+								<md-card-title>
+									<md-card-title-text>
+										<span>Disabled GQL Queries</span>
+									</md-card-title-text>
+								</md-card-title>
+								<md-card-content>
+									<div>
+										<md-input-container class="control" style="width: 200px;">
+											<label>Operation name to disable</label>
+											<input type="text" default="ABC" ng-model="queryMapping.newQuery" />
+										</md-input-container>
+										<md-button ng-click="addDisabledGQLQuery()">
+											<i class="fa fa-plus"></i>
+										</md-button>
+										<div id="ec2-list" class="form-group" ng-repeat="(index, queryName) in Settings.disabled_gql_queries">
+											<section layout="row" flex>
+												<div>
+													<span>[[queryName]]</span>
+													<button class="btn btn-default btn-danger" type="button" ng-click="removeDisabledGQLQuery(queryName)">
+														<i class="fa fa-trash"></i>
```

Comment:
Can we move the trashcan to be in a consistent column (rather than right next to the value)? At least that's how it's looking in the screenshot.